keps/sig-network/784-kube-proxy-component-config/README.md

```diff
@@ -271,17 +365,15 @@ No.
 
 ###### Will enabling / using this feature result in introducing new API types?
 
-Yes.
-[WIP]
+No.
```

Comment:
Hm... I'm not sure if this question is meant to include config API types (in which case the answer would be "yes")

Reply:
Updated, taking [this](https://github.com/kubernetes/enhancements/pull/1996/files#diff-bfe309d572be63a952d0253bdc7b0a9b1820f667fa698b69f66df53a6097f71aR501) as reference.